src/components/Clock/Clock.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import './Clock.css';
+import moment from 'moment';
+
+class Clock extends React.Component {
+  state = {
```

Comment:
Trodde ikke det var lov å sette state uten konstruktør, men ser at det er helt greit.

> The syntax is not widely adopted yet, because class field declaration are a new feature in JavaScript, but once they will be supported by all browsers, they may be used more widely in React to create initial state in a class component.

(Vi bruker noe som heter babel som gjør at all kode vi skriver vil gjøres om slik at det funker i eldre browsere også. Så det som står rett over om at det kun funker i nye browsere stemmer ikke for oss (tror jeg))

Reply:
Notert!